src/datastore/firebase.ts

```diff
@@ -40,8 +46,10 @@ initializeFirestore(app, {
 const auth = getAuth();
 
 // Sign up handler
-export const signUp = (email: string, password: string) =>
+export const signUp = (email: string, password: string) => {
 	createUserWithEmailAndPassword(auth, email, password);
```

Comment:
Nejde mi to pustit, failuje to na ` Uncaught (in promise) FirebaseError: Missing or insufficient permissions.`
Jak máš nastavená práva ve firebase?

Možná vím, v čem je problém, mám povolené allow read/write jen pro autentizované, ale čekal bych, že je to dřív přihlásí a až pak ukládá?

Reply:
To zní, že by mohlo být ono! Problém jsem s tím ale neměl.

Reply:
A máš to nastavené jak? Kdokoliv read, write?

Reply:
```
rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if true;
    }
  }
}
```